### PR TITLE
feat(query): added "EKS cluster logging is not enabled" query for Terraform

### PR DESCRIPTION
**Pull request by felipe-avelar**
Signed-off-by: Felipe Avelar <felipe.avelar@checkmarx.com>

**Proposed Changes**
- Added "EKS cluster logging is not enabled" query for Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/eks_cluster_log_disabled/test/negative1.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+resource "aws_eks_cluster" "negative1" {
+  depends_on = [aws_cloudwatch_log_group.example]
+
+  enabled_cluster_log_types = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
+  name                      = var.cluster_name
+
+  # ... other configuration ...
+}
+
+resource "aws_cloudwatch_log_group" "negative2" {
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/eks_cluster_log_disabled/test/negative1.tf:15]: split the samples

File: assets/queries/terraform/aws/eks_cluster_log_disabled/test/positive_expected_result.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "queryName": "EKS cluster logging is not enabled",
+    "severity": "LOW",
+    "line": 1,
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/eks_cluster_log_disabled/test/positive_expected_result.json:5]: unit tests are failing

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/aws/eks_cluster_log_disabled/test/negative1.tf (hunk 1 above)
Comment (felipe-avelar) [reply assets/queries/terraform/aws/eks_cluster_log_disabled/test/negative1.tf:15]: The snippets are differents, but we don't need this part on negative example, will remove it

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**